common/artifact-manager/pom.xml

```diff
@@ -131,6 +131,11 @@
 			<artifactId>commons-io</artifactId>
 			<version>2.7</version>
 		</dependency>
+		<dependency>
+                        <groupId>org.apache.commons</groupId>
+                        <artifactId>commons-collections4</artifactId>
```

Comment:
Just for my info, what is this one being used for

Reply:
It is an external library provided by Apache with various collection utilities classes and methods (like union, subtract, intersection, etc.)